src/components/commons/repository/RepositoryCard.js

```diff
@@ -11,27 +12,39 @@ type RepositoryProps = {
 type StatProps = {
     iconClass: string,
     statCount: number,
+    tooltipMessage: String,
 };
 
-const Stat = ({ iconClass, statCount }: StatProps) => (
-    <div className="stat">
+const Stat = ({ iconClass, statCount, tooltipMessage }: StatProps) => (
+    <div className="stat" data-tip={tooltipMessage}>
         <i className={`fas ${iconClass}`} />
         <span className="stat-count">{statCount}</span>
     </div>
 );
 
 const RepositoryStats = ({ repository }: RepositoryProps) => (
     <div className="repository-stats">
-        <Stat iconClass="fa-star" statCount={repository.stargazersCount} />
+        <Stat
+            iconClass="fa-star"
+            statCount={repository.stargazersCount}
+            tooltipMessage="Estrelas"
+        />
         <Stat
             iconClass="fa-exclamation-circle"
             statCount={repository.issuesCount}
+            tooltipMessage="Issues"
         />
         <Stat
             iconClass="fa-code-branch"
             statCount={repository.pullRequestsCount}
+            tooltipMessage="Pull Requests"
+        />
+        <Stat
+            iconClass="fa-utensils"
+            statCount={repository.forkCount}
+            tooltipMessage="Forks"
```

Comment:
Nesses campos `tooltipMessage` está dando o erro `Cannot create Stat element because string [1] is incompatible with String [2] in property tooltipMessage.`.

Tu poderia tentar verificar o que seja? Se não conseguir, comenta aqui que posso divulgar com você